e2e/cypress/integration/integrations/integrations_page_spec.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+describe('Integrations', () => {
+    let teamA;
+
+    before(() => {
+        // # Login, create incoming webhook for Team A
```

Comment:
This seems to be not accurate. How about?
```suggestion
        // # Setup with the new team
```